web/src/composebox_typeahead.js

```diff
@@ -1141,7 +1141,7 @@ export function initialize_compose_typeahead(selector) {
         // O(n) behavior in the number of users in the organization
         // inside the typeahead library.
         source: get_sorted_filtered_items,
-        highlighter: content_highlighter,
+        highlighter_html: content_highlighter,
```

Comment:
Rename `content_highlighter` to `content_highlighter_html`. (The goal with all these renames is to establish a local chain of reasoning for safety. We want any future bug that might introduce unsafe HTML into a value to be detectable without referencing a different function to see where the value is ultimately used.)

Reply:
done :)